ci/utils.groovy

```diff
@@ -69,7 +69,7 @@ def pkgFilename(type, ext) {
 
 def doGitRebase() {
   /* rebasing on relases defeats the point of having a release branch */
-  if (params.BUILD_TYPE == 'release') {
+  if (params.BUILD_TYPE == 'release' || branchName() == 'canary-branch') {
     println 'Skipping rebase due to release build...'
```

Comment:
Maybe we could add `params.BUILD_TYPE`/`branchName()` to the message instead of `release`?